### ISSUE TITLE
border-collapse other than "collapse" is stripped, even when safelisted

### ISSUE THREAD

**Issue by bbugh** [flavorjones/loofah#201]
Any valid properties for `border-collapse` except `collapse` is being stripped out; `initial`, `unset`, `inherit`, etc. This makes it impossible to do `border-radius` on tables that have a stylesheet default of `collapse`.

This happens with all of the scrubbers (that I know of) that would affect this: `prune`, `strip`, `whitewash`.

Example:

```ruby
>> Loofah.fragment(%Q(<table style="border-collapse: separate;"><tr><td></td></tr></table)).scrub!(:prune).to_s
=> "<table><tr><td></td></tr></table>"
```

But:

```ruby
>> Loofah::HTML5::SafeList::ALLOWED_CSS_PROPERTIES.include?('border-collapse')
=> true
```

**Comment by flavorjones** [comment]
@bbugh Thanks for opening this issue, and sorry for my slow reply. I've reproduced this and it's because `collapse` is a member of `Loofah::HTML5::SafeList::ACCEPTABLE_CSS_KEYWORDS`.

See #214 for the fix.

**Comment by bbugh** [comment]
Thank you! No problem, you're a champion for all the work you do to keep this running.

**Comment by flavorjones** [comment]
Fixed in v2.11.0
